native/python/sokuji_native/_ffi.py

```
@@ -0,0 +1,41 @@
+class sk_init_options(ctypes.Structure):
+    _fields_ = [("abi_version", c_int32), ("n_threads", c_int32), ("module_dir", c_char_p),
+                ("log", LOG_CB), ("log_user", c_void_p)]
+
+
+class sk_device(ctypes.Structure):
+    _fields_ = [("index", c_int32), ("kind", c_int32), ("name", c_char * 64), ("description", c_char * 128),
+                ("mem_total", c_uint64), ("mem_free", c_uint64)]
+
+
+def bind(lib: ctypes.CDLL) -> ctypes.CDLL:
+    lib.sk_init.argtypes = [POINTER(sk_init_options)];           lib.sk_init.restype = c_int32
```

Ruff (0.16.2)

[error] Multiple statements on one line (semicolon)

(E702)